sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample00_AuthenticateClient.md

```diff
@@ -0,0 +1,56 @@
+# Authenticate the client
+
+This sample demonstrates how to authenticate the `ServiceBusClient`, which is the starting point for all interactions with the Service Bus client library.
+
+## Authenticate with a connection string
+
+The simplest way to authenticate with the Service Bus is to use a connection string, which is created automatically when creating a Service Bus namespace. If you aren't familiar with connection strings in Service Bus, you may wish to follow the step-by-step guide to [get a Service Bus connection string](https://docs.microsoft.com/azure/service-bus-messaging/service-bus-quickstart-topics-subscriptions-portal#get-the-connection-string). For production scenarios, we recommend using Azure.Identity authentication as it provides [role-based access control](https://docs.microsoft.com/azure/role-based-access-control/overview) without the need to manage connection strings or shared access keys.
```

Comment:
```suggestion
The simplest way to authenticate with Service Bus is to use a connection string, which is created automatically when creating a Service Bus namespace. If you aren't familiar with connection strings in Service Bus, you may wish to follow the step-by-step guide to [get a Service Bus connection string](https://docs.microsoft.com/azure/service-bus-messaging/service-bus-quickstart-topics-subscriptions-portal#get-the-connection-string). For production scenarios, we recommend using Azure.Identity authentication as it provides [role-based access control](https://docs.microsoft.com/azure/role-based-access-control/overview) without the need to manage connection strings or shared access keys.
```
This might be totally fine, I just haven't read service bus referred to as the service bus way before